prism-languages.php

```diff
@@ -5,7 +5,7 @@
  * @return array Hash of lang => language
```

Comment:
:no_entry_sign: **Error**: There must be exactly one blank line before the tags in a doc comment (*Generic.Commenting.DocComment.SpacingBeforeTags*).